Automattic/simplenote-macos
# Fastlane `match`: Migrate Code Signing from Google Cloud to S3

### Summary
(I was just going to update the profiles that had expired, but decided to update these lanes while I was at it)

This PR: 
- Migrates the code signing certificates and provisioning storage from Google Cloud to S3
- Updates Fastlane from `2.219.0` to `2.220.0`
- Updates the needed values in the example user `env` file
- Removes unneeded files from `.configure` - We don't need the Google or Apple API auth files because this PR switches to using S3 and the `app_store_connect_api_key` Fastlane action. By using the `app_store_connect_api_key` action, it'll make rotating the ASC key much easier because it's just pulling from the shared file in Mobile Secrets. 

### Test
1. In your local `~/.simplenotemac-env.default` env file, add values for `MATCH_S3_ACCESS_KEY`, `MATCH_S3_SECRET_ACCESS_KEY`, and `MATCH_PASSWORD` from Mobile Secrets
2. Run `bundle exec fastlane app_store_code_signing`
3. Verify that it completes successfully
4. Run `bundle exec fastlane app_store_code_signing readonly:false`
5. Verify that it completes successfully (everything is up to date, so no changes will be made)

### Review


### Release
These changes do not require release notes.
